src/services/legal-services.ts

```diff
@@ -370,7 +360,7 @@ export default class LegalServices {
       }
     }
 
-    return axios.put(url, file, merge(this.config, extraConfig))
```

Comment:
No longer need to merge the module config with the extra config -- the module config is handled in AxiosInstance and the extra config can be simply passed in here (it's all merged internally).